content/features/00-graph-overview.adoc

```diff
@@ -3,10 +3,10 @@ title: "Graph"
 date: 2018-06-20T19:04:38+02:00
 draft: false
 image: features/graph-overview.png
-subTitle: "See how your microservices are connected in the Kiali graph"
+subTitle: "Gain insight into how your mesh is functioning"
 weight: -1
 ---
 
-The graph view allows you to select a time period and see the **requests per second** between your microservices, the **response time** of those requests and whether they were secured with **mutual TLS**.
+The graph provides a powerful way to be able to visualize what is actually happening within your service mesh. This allows you to know what components are **communicating** with each other, to see where **errors** are occurring and how **healthy** your components are, to know if where things like routerules or circuit breakers are **configured**, and to understand the rate of your **traffic flow** and other **metrics**.
 
-The graph view **groups** applications by version, this makes it easy to see how each version of the application is behaving.
+All while providing a convenient and powerful configuration options to be able to get to the data that is important to you and visualized it in a manner that you desire.
```

Comment:
"All while providing" is a bit strange to start the sentence with. 
Which powerful configuration options are provided atm?

Reply:
OK, I just understood configuration as configuring Istio, not different isplay options of data on the graph